sql/complete-schema.sql

```diff
@@ -1,581 +1,1641 @@
-    `labelType`              varchar(255) DEFAULT NULL,
-    `lcColumn`               varchar(255) DEFAULT NULL,
-    `lcmsMethod`             varchar(255) DEFAULT NULL,
-    `measurementCode`        varchar(255) DEFAULT NULL,
-    `IRI`                    varchar(255) DEFAULT NULL,
-    `label`                  varchar(255) DEFAULT NULL,
-    `projectId`              varchar(255) DEFAULT NULL,
-    `registration`           datetime(6)  DEFAULT NULL,
-    `samplePool`             varchar(255) DEFAULT NULL,
-    `technicalReplicateName` varchar(255) DEFAULT NULL,
-    PRIMARY KEY (`measurement_id`)
-) ENGINE = InnoDB
-  DEFAULT CHARSET = utf8mb4
-  COLLATE = utf8mb4_unicode_ci;
+    `measurement_id`
+    varchar
+(
+    255
+) NOT NULL,
+    `digestionEnzyme` varchar
+(
+    255
+) DEFAULT NULL,
+    `digestionMethod` varchar
+(
+    255
+) DEFAULT NULL,
+    `enrichmentMethod` varchar
+(
+    255
+) DEFAULT NULL,
+    `facility` varchar
+(
+    255
+) DEFAULT NULL,
+    `injectionVolume` int
+(
+    11
+) DEFAULT NULL,
+    `instrument` longtext DEFAULT NULL CHECK
+(
+    json_valid
+(
+    `instrument`
+)),
+    `labelType` varchar
+(
+    255
+) DEFAULT NULL,
+    `lcColumn` varchar
+(
+    255
+) DEFAULT NULL,
+    `lcmsMethod` varchar
+(
+    255
+) DEFAULT NULL,
+    `measurementCode` varchar
+(
+    255
+) DEFAULT NULL,
+    `IRI` varchar
+(
+    255
+) DEFAULT NULL,
+    `label` varchar
+(
+    255
+) DEFAULT NULL,
+    `projectId` varchar
+(
+    255
+) DEFAULT NULL,
+    `registration` datetime
+(
+    6
+) DEFAULT NULL,
+    `samplePool` varchar
+(
+    255
+) DEFAULT NULL,
+    `technicalReplicateName` varchar
+(
+    255
+) DEFAULT NULL,
+    PRIMARY KEY
+(
+    `measurement_id`
+)
+    ) ENGINE = InnoDB
+    DEFAULT CHARSET = utf8mb4
+    COLLATE = utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `purchase_offer`
 (
-    `id`           bigint(20) NOT NULL AUTO_INCREMENT,
-    `file_content` longblob     DEFAULT NULL,
-    `fileName`     varchar(255) DEFAULT NULL,
-    `signed`       bit(1)     NOT NULL,
-    PRIMARY KEY (`id`)
-) ENGINE = InnoDB
-  AUTO_INCREMENT = 1
-  DEFAULT CHARSET = utf8mb4
-  COLLATE = utf8mb4_unicode_ci;
+    `id`
+    bigint
+(
+    20
+) NOT NULL AUTO_INCREMENT,
+    `file_content` longblob DEFAULT NULL,
+    `fileName` varchar
+(
+    255
+) DEFAULT NULL,
+    `signed` bit
+(
+    1
+) NOT NULL,
+    PRIMARY KEY
+(
+    `id`
+)
+    ) ENGINE = InnoDB
+    AUTO_INCREMENT = 1
+    DEFAULT CHARSET = utf8mb4
+    COLLATE = utf8mb4_unicode_ci;
 
 CREATE TABLE IF NOT EXISTS `quality_control_upload`
 (
-    `id`            bigint(20) NOT NULL AUTO_INCREMENT,
-    `experiment_id` varchar(255) DEFAULT NULL,
-    `file_content`  longblob     DEFAULT NULL,
-    `fileName`      varchar(255) DEFAULT NULL,
-    PRIMARY KEY (`id`)
-) ENGINE = InnoDB
-  AUTO_INCREMENT = 1
-  DEFAULT CHARSET = utf8mb4
-  COLLATE = utf8mb4_unicode_ci;
+    `id`
```

Comment:
could you please undo this formatting. This is not java and I think this formatting is way harder to read.